quotes/out/cldr.inc

```diff
@@ -9,21 +9,21 @@
 :root:lang(az),       :not(:lang(az)) > :lang(az)             { quotes: '\201c' '\201d' '\2018' '\2019' } /* &#x201c; &#x201d; &#x2018; &#x2019; */
 :root:lang(az-Cyrl),  :not(:lang(az-Cyrl)) > :lang(az-Cyrl)   { quotes: '\00ab' '\00bb' '\2039' '\203a' } /* &#x00ab; &#x00bb; &#x2039; &#x203a; */
 :root:lang(bas),      :not(:lang(bas)) > :lang(bas)           { quotes: '\00ab' '\00bb' '\201e' '\201c' } /* &#x00ab; &#x00bb; &#x201e; &#x201c; */
-:root:lang(be),       :not(:lang(be)) > :lang(be)             { quotes: '\00ab' '\00bb' '\00ab' '\00bb' } /* &#x00ab; &#x00bb; &#x00ab; &#x00bb; */
+:root:lang(be),       :not(:lang(be)) > :lang(be)             { quotes: '\00ab' '\00bb' '\201e' '\201c' } /* &#x00ab; &#x00bb; &#x201e; &#x201c; */
```

Comment:
:root:lang(be),       :not(:lang(be)) > :lang(be)             { quotes: '\00ab' '\00bb' '\201e' '\201c' } /* &#x00ab; &#x00bb; &#x201e; &#x201c; */

That one surprised me when I saw it and made we wonder if we have a bug somewhere. But then I notice it aligns `be` with the data we already have here for `ru` and `uk`, which is how it should be.